pwa/src/layout/Layout.tsx

```diff
@@ -86,19 +84,13 @@ const Layout: React.FC<LayoutProps> = ({ children, pageContext, location }) => {
       <GatsbyProvider value={gatsbyContext}>
         <APIProvider value={API}>
           <FiltersProvider value={[filters, setFilters]}>
-            <ThemesContextProvider value={[themeContext, setThemeContext]}>
-              <ThemeProvider>
-                <ToolTip id={TOOLTIP_ID} />
-
-                <Favicon url={Logo} />
-
-                <HeaderTemplate layoutClassName={styles.header} />
-
-                <div className={styles.pageContent}>{children}</div>
-
-                <FooterTemplate layoutClassName={styles.footer} />
-              </ThemeProvider>
-            </ThemesContextProvider>
+            <Document>
+              <ToolTip id={TOOLTIP_ID} />
+              <Favicon url={Logo} />
+              <HeaderTemplate layoutClassName={styles.header} />
+              <div className={styles.pageContent}>{children}</div>
+              <FooterTemplate layoutClassName={styles.footer} />
```

Comment:
I'd opt to keep some white space in between these lines, as they're doing separate things.